service/service.go

```diff
@@ -167,11 +225,7 @@ func New(config Config) (*Service, error) {
 			IncludeTags:                config.Viper.GetBool(config.Flag.Service.AWS.IncludeTags),
 			InstallationName:           config.Viper.GetString(config.Flag.Service.Installation.Name),
 			IPAMNetworkRange:           ipamNetworkRange,
-			LabelSelector: controller.ClusterConfigLabelSelector{
-				Enabled:          config.Viper.GetBool(config.Flag.Service.Feature.LabelSelector.Enabled),
-				OverridenVersion: config.Viper.GetString(config.Flag.Service.Test.LabelSelector.Version),
-			},
```

Comment:
@kopiczko @TheoBrigitte do you know what this is all about? I couldn't make sense of it and was wondering why we added it. It did not seem to do anything.

Reply:
Leftover. Kill with 🔥

Reply:
Oh I am so good at that fire thing. 🙈

Reply:
Too good.